discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -10,7 +17,8 @@
 from test_infra.controllers.node_controllers.node_controller import NodeController
 
 
-class LibvirtController(NodeController):
+class LibvirtController(NodeController, ABC):
```

Comment:
It seems like there are lots of disk functions.. Maybe we can separate it somehow logically? inner-class?

Reply:
That thought crossed my mind, I personally think adding another abstraction / layer / class here will make things more confusing/complicated than just simply having those methods inside the class. They do logically fit here.